pre-commit-config.yaml

```diff
@@ -5,10 +5,17 @@ repos:
     rev: v3.2.0
     hooks:
     -   id: trailing-whitespace
-        exclude: "^secrets/|environments/.*/secrets.cfg"
+        exclude: |
+          (?x)^(
+            secrets/|environments/.*/secrets\.cfg|
+            .*\.patch
+          )$
     -   id: end-of-file-fixer
-        # These are non-anchored regular rexpressions
```

Comment:
@ctheune I'd like to get your feedback on this "non-anchored re" aspect, as you were the one to introduce this comment.

Isn't `^` an anchor as well? Because that character has been present even when you had introduced that comment.

Reply:
This was more a reminder that exclude is not anchored by default